src/modules/sensors/vehicle_gps_position/VehicleGPSPosition.hpp

```diff
@@ -52,8 +52,12 @@
 #define BLEND_MASK_USE_VPOS_ACC     4
 
 // define max number of GPS receivers supported and 0 base instance used to access virtual 'blended' GPS solution
-#define GPS_MAX_RECEIVERS 2
-#define GPS_BLENDED_INSTANCE 2
+#define GPS_MAX_RECEIVERS 3
```

Comment:
This does cause more RAM to be used, so it may be a good idea to allow a board-specific cmake file to either set this directly or indirectly (i.e., if there's a CONSTRAINED_RAM #define, set this to 2, otherwise set to 4, for example).

In any case, this should probably be set back to 2 before merging, and allow developers to increase this on their own if desired (for now).

Reply:
Changed 3 -> 2 for now. Easy to change later.